packages/cascara/src/ui/Columns/Columns.js

```diff
@@ -21,6 +21,8 @@ const propTypes = {
   style: pt.shape(pt.object),
 };
 
+const componentName = 'Columns';
+
```

Comment:
I don't see anywhere else in the component where we are using the `componentName` const. To me this feels like extra work for something that we should not be considering a configurable global. This data attribute should always be the name of the component itself. I think we should just put these strings directly on the `data-component='Columns'` attribute.